lib/src/rules/prefer_for_elements_to_map_fromIterable.dart

```diff
@@ -13,6 +13,15 @@ const _desc = r'Prefer for elements when building maps from iterables.';
 const _details = r'''
 When building maps from iterables, it is preferable to use for elements.
 
+Using for elements brings several benefits including:
```

Comment:
The "for" in "for elements" here reads a little too much as a separate word, not part of the "for elements". Makes me thing "use *what* for *which* elements?".

Consider quoting ("for elements") or dashing (for-elements) or even code-quoting the "for" (`for`-elements), both here and above.

Reply:
Agree, I was following the existing phrasing but it doesn't sound great to me either.

"for elements" itself sounds a bit odd to me, but maybe others are used to that.. I would've thought something like "Prefer for-loops when building maps from iterables.", or is "elements" preferred?